src/scicookie/{{cookiecutter.project_slug}}/build-system/poetry-pyproject.toml

```diff
@@ -112,71 +82,5 @@ pandoc = "^2.3"
 jupyter-book = "^0.15.1"
 myst-parser = "^0.18.1"
 {% endif %}
-{%- if cookiecutter.use_pytest == "yes" %}
-[tool.pytest.ini_options]
-testpaths = [
-    "tests",
-]
-{% endif %}
-{%- if cookiecutter.use_blue == "yes" %}
-[tool.blue]
-line-length = 79
-target-version = ["py38"]
-force-exclude = '''(?x)(
-    docs/*
-  | .*\\.egg-info
-)'''  # TOML's single-quoted strings do not require escaping backslashes
-{% endif %}
-{%- if cookiecutter.use_black == "yes" %}
-[tool.black]
-line-length = 79
-target-version = ["py38"]
-force-exclude = '''(?x)(
-    docs/*
-  | .*\\.egg-info
-)'''  # TOML's single-quoted strings do not require escaping backslashes
-{% endif %}
-{%- if cookiecutter.use_isort == "yes" %}
-[tool.isort]
-ensure_newline_before_comments = true
-line_length = 79
-multi_line_output = 3
-include_trailing_comma = true
-skip_glob = ["docs/*", "*.egg-info"]
-{% endif %}
-{%- if cookiecutter.use_bandit == "yes" %}
-[tool.bandit]
-exclude_dirs = ["tests"]
-targets = "{{ package_path }}"
-{% endif %}
-{%- if cookiecutter.use_vulture == "yes" %}
-[tool.vulture]
-exclude = ["tests"]
-ignore_decorators = []
-ignore_names = []
-make_whitelist = true
-min_confidence = 80
-paths = ["{{ package_path }}"]
-sort_by_size = true
-verbose = false
-{% endif %}
-{%- if cookiecutter.use_ruff == "yes" %}
-[tool.ruff]
-line-length = 79
-force-exclude = true
-src = ["{{ package_path }}"]
-exclude = [
-  'docs',
-]
-select = [
-  "F",   # pyflakes
-]
 
-[tool.ruff.pydocstyle]
-convention = "numpy"
-{% endif %}
-{%- if cookiecutter.use_mypy == "yes" %}
-[tool.mypy]
-no_strict_optional = false
-{% endif %}
-{#- keep this line at the end of the file -#}
```

Comment:
@xmnlab I think this line should go at the end of the file, there is no empty line at the end. That's what I can see

Reply:
thanks @YurelyCamacho , I will revisit that again now! appreciate your review!